typescript/examples/langchain-cdp-chatbot/chatbot.ts

```diff
@@ -122,6 +123,9 @@ async function initializeAgent() {
             ]
           : []),
         alloraActionProvider(),
+        flaunchActionProvider({
```

Comment:
Please remove the flaunch action from this list of action providers in the example